apm-sniffer/apm-sdk-plugin/http-plugin/okhttp-3.x-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -0,0 +1 @@
+org.skywalking.apm.plugin.okhttp.v3.define.RealCallInstrumentation
```

Comment:
The plugin define should be like this:
`PluginName_PluginVersion= Instrumentation class`